extension/src/test/suite/plots/data/index.test.ts

```diff
@@ -211,13 +214,38 @@ suite('Plots Data Test Suite', () => {
 
       const managedUpdateSpy = spy(data, 'managedUpdate')
       const dataUpdatedEvent = new Promise(resolve =>
-        data.onDidUpdate(() => resolve(undefined))
+        disposable.track(data.onDidUpdate(() => resolve(undefined)))
       )
 
       await fireWatcher(join(dvcDemoPath, watchedFile))
       await dataUpdatedEvent
 
-      expect(managedUpdateSpy).to.be.called
+      expect(
+        managedUpdateSpy,
+        'should update data when an event is fired for a watched file'
+      ).to.be.called
+      managedUpdateSpy.resetHistory()
+
+      bypassProcessManagerDebounce(mockNow, 2)
+
+      const secondDataUpdatedEvent = new Promise(resolve =>
+        disposable.track(data.onDidUpdate(() => resolve(undefined)))
+      )
+
+      const absParentDirectory = join(dvcDemoPath, parentDirectory)
+
+      if (getProcessPlatform() === 'win32') {
+        removeDir(absParentDirectory)
```

Comment:
`utimes` will not update a directory on Windows (assumption from the test failing). I have gone with this destructive action for now. If there are ever any issues we can come back and skip this test for Windows instead.